weave-js/src/integrations.ts

```diff
@@ -0,0 +1,73 @@
+// Direct copy of the Datadog integration from the Weights & Biases codebase
+// Minimal implementation of RUM logging
+
+import {datadogRum} from '@datadog/browser-rum';
+import {once} from 'lodash';
+
+declare global {
+  interface Window {
+    DatadogEnabled?: boolean;
+  }
+}
+
+const DATADOG_RUM_APPLICATION_ID = 'ddeeb29c-5e8c-4579-90be-f7c0cc91dbcd';
```

Comment:
Just to double check - this value should be the same as the app's?

Reply:
Fixed